apps/src/applab/reducers.js

```diff
@@ -38,7 +39,8 @@ function level(state, action) {
         'isEmbedView',
         'isReadOnlyWorkspace',
         'isShareView',
-        'isViewDataButtonHidden'
+        'isViewDataButtonHidden',
+        'instructionsMarkdown'
```

Comment:
Putting it here makes sense to me, but points out how some of our names are confusing.  Your instructions reducer seems concerned with the instruction pane state, not its contents; where the contents are a property of the level.